src/data/markdown/docs/01 guides/02 Using k6/14 Scenarios/03 Graceful stop.md

```diff
@@ -0,0 +1,64 @@
+---
+title: 'Graceful stop'
+excerpt: ''
+---
+
+> ### 🎉 New in v0.27.0
+>
+> This feature is new as of version 0.27.0. Usage of this feature is optional and for the vast majority,
+> existing scripts and configurations will continue to work as before. For a list of breaking changes,
+> see the [k6 v0.27.0 release notes](https://github.com/loadimpact/k6/releases/tag/v0.27.0)).
+
+## Description
+
+This option is available for all executors except `externally-controlled` and allows the
```

Comment:
I think we should add some background here about how k6 stopped iterations before v0.27.0, to explain why these options are useful.

Something like:

Prior to v0.27.0, k6 would interrupt any iterations in progress when the test duration was reached or when scaling down VUs with the `stages` option. In some cases this would lead to skewed metrics and unexpected test results. In v0.27.0 this behavior can be controlled with the `gracefulStop` and `gracefulRampDown` options.